lib/vehicle.rb

```diff
@@ -5,14 +5,21 @@ class Vehicle
               :year,
               :make,
               :model,
-              :engine
+              :engine,
+              :registration_county
+  attr_accessor :registration_date, :plate_type
```

Comment:
For the purposes of this project, it is totally okay to use an attr_accessor. However, we often want to stay away from these and instead create methods for changing the value of our attributes. We want to be able to have control over these data changes, and maintain data integrity. Right now, someone could change the registration_date to `banana` and we have no control over that. But, if you added a "setter" method in this class that an outside class had to call, like `set_registration_date(date)` then you could add logic like checking that it's a valid date. This is a good rule of thumb for adhering to the principle of encapsulation in our our classes.